packages/react-component-library/src/components/RangeSliderE/RangeSliderE.stories.tsx

```diff
@@ -0,0 +1,171 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Story, Meta } from '@storybook/react'
+
+import {
+  IconBrightnessLow,
+  IconBrightnessHigh,
+} from '@defencedigital/icon-library'
+import { RangeSliderE, RangeSliderEProps } from './index'
+
+export default {
+  component: RangeSliderE,
+  title: 'Range Slider (Experimental)',
+  parameters: {
+    actions: { argTypesRegex: '^on.*' },
+  },
+} as Meta
+
+const disableColorContrastRule = {
+  a11y: {
+    config: {
+      rules: [
+        {
+          id: 'color-contrast',
+          enabled: false,
+        },
+      ],
+    },
+  },
+}
+
+const StyledWrapper = styled.div`
+  display: flex;
+  height: 5rem'
```

Comment:
May need to bump the top-padding for the canvas view:

![image](https://user-images.githubusercontent.com/66470099/144025690-87b2e031-c7a7-4a7f-9a7d-d1b3173c5800.png)